js-commit-guidelines.md

```diff
@@ -0,0 +1,99 @@
+# Git Commit Guidelines
+
+We have very precise rules over how our git commit messages can be formatted.
+This leads to **more readable messages** that are easy to follow when
+looking through the **project history**.  But also,
+we use the git commit messages to **generate the change log**.
```

Comment:
I don't feel that it is important to keep the emphasis. My reasoning is that I find it comes from a standpoint of lecturing, saying that this is how we do things. I think that because the emphasis is pseudo-random it also comes off as emotional - these words are important for me, as a writer, so I am going to make sure that I get them across. 

I think it is more in the spirit of collaboration, and trusting the reader, not to put in these. I know this is a stylistic choice - @jbenet, for instance, _really_ likes to use italics, bold, and headings as often as he can. In a styleguide, though, I think that it is important to nail this, as people will see this as a formal declaration, and any emotive writing that could be taken as aggressive, condescending, or bland needs to be avoided.